src/main/kotlin/no/nav/tilleggsstonader/sak/vilkår/stønadsvilkår/VilkårService.kt

```diff
@@ -296,6 +296,17 @@ class VilkårService(
         vilkårRepository
             .findByBehandlingId(behandlingId)
             .filter { it.resultat == Vilkårsresultat.OPPFYLT }
+            .also { it.kstFeilHvisTypeIkkeGlederForDagligReise() }
```

Comment:
Ser at det ikke er noe du innfører, da det samme gjøres over for boutgifter. Men jeg synes det var et litt overraskende sted å validere dette, burde det ikke holde at det gjøres ved lagring av vilkår? 🤔